panda_moveit_config/config/panda_gripper_controllers.yaml

```diff
@@ -0,0 +1,19 @@
+controller_list:
+  - name: position_joint_trajectory_controller
+    action_ns: follow_joint_trajectory
+    type: FollowJointTrajectory
+    joints:
+      - panda_joint1
+      - panda_joint2
+      - panda_joint3
+      - panda_joint4
+      - panda_joint5
+      - panda_joint6
+      - panda_joint7
+  - name: franka_gripper
+    action_ns: gripper_action
+    type: GripperCommand
+    parallel: true
```

Comment:
Where is the documentation for this? I don't see it in [ros_controllers](https://github.com/ros-controls/ros_controllers)

Reply:
I found it in http://docs.ros.org/lunar/api/moveit_simple_controller_manager/html/gripper__controller__handle_8h_source.html, where it's explained on line 205. Not sure if it's documented somewhere else.

Reply:
Nice! I hadn't seen that actually:

https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_plugins/moveit_simple_controller_manager/src/moveit_simple_controller_manager.cpp#L113

@mlautman please document on [this tutorial](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/pr2_tutorials/planning/src/doc/controller_configuration.html) under GripperCommands

Reply:
will do